src/components/SUE/report/SueReportLocation.tsx

```diff
@@ -48,6 +57,27 @@ export const SueReportLocation = ({
   const zipCodeInputRef = useRef();
   const cityInputRef = useRef();
 
+  const queryVariables = {
+    status:
+      'TICKET_STATUS_UNPROCESSED,TICKET_STATUS_OPEN,TICKET_STATUS_IN_PROCESS,TICKET_STATUS_WAIT_REQUESTOR,TICKET_STATUS_WAIT_THIRDPARTY'
+  };
+
+  const { data, isLoading } = useQuery([QUERY_TYPES.SUE.REQUESTS, queryVariables], () =>
+    getQuery(QUERY_TYPES.SUE.REQUESTS)(queryVariables)
+  );
+
+  const mapMarkers = useMemo(
+    () =>
+      mapToMapMarkers(
+        parseListItemsFromQuery(QUERY_TYPES.SUE.REQUESTS, data, undefined, {
```

Comment:
why not `QUERY_TYPES.SUE.REQUESTS_WITH_SERVICE_REQUEST_ID` as for the markers on the map?

Reply:
If I use `QUERY_TYPES.SUE.REQUESTS_WITH_SERVICE_REQUEST_ID`, only the marker for the report with a specific ID will be shown. We need to show markers for all data.

Reply:
Ow, I'm sorry. I misunderstood. It's just a parser. They both parse the same data. We can use the `REQUESTS_WITH_SERVICE_REQUEST_ID` query to parse.

Reply:
Ok I thought of that, but wasn't sure. Yes, let's have it in the same way in both places.